### PR TITLE
Lab13

### PR DESCRIPTION
**Pull request by afonyaa**

### CODE REVIEW

**Comment by none-word** [comment]
Amazing!

File: k8s/Chart/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
```
Comment (AKurmazov) [inline k8s/Chart/templates/deployment.yaml:1]: Nit: You should rename this file to `statefulset.yaml`

**Comment by AKurmazov** [comment]
👍

**Comment by Maxkoz777** [comment]
Good job!

**Comment by Homa3030** [comment]
👍🏻

**Pull request closed without merging**